address-space-controller/src/main/java/io/enmasse/controller/PodDisruptionBudgetController.java

```diff
@@ -17,13 +17,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class PodDistruptionBudgetController implements Controller {
-    private static final Logger log = LoggerFactory.getLogger(PodDistruptionBudgetController.class);
+public class PodDisruptionBudgetController implements Controller {
+    private static final Logger log = LoggerFactory.getLogger(PodDisruptionBudgetController.class);
 
     private final NamespacedKubernetesClient client;
     private final String namespace;
 
-    public PodDistruptionBudgetController(NamespacedKubernetesClient client, String namespace) {
+    public PodDisruptionBudgetController(NamespacedKubernetesClient client, String namespace) {
```

Comment:
It looks like namespace is tied to the client? If so, then `namespace` parameter seems unnecessary.

Reply:
For some reason, kubernetes-client tends to complain about operations missing the namespace if it is not supplied. I could've also used client.getNamespace() I guess, but I'd rather change all our controllers in one go for that.